docs/src/upgrade/release-3.7.x.asciidoc

```diff
@@ -29,7 +29,10 @@ complete list of all the modifications that are part of this release.
 
 === Upgrading for Users
 
+==== String Manipulation Steps
```

Comment:
this reads a bit too much like a CHANGELOG entry to me - a few more words I think are in order. New steps in upgrade docs should have some examples. i'd also include links in "See" to the reference docs. 

also, please add provider docs for these steps.

Reply:
Updated